### DISCUSSION TITLE
How are Organic maps transit lines this beautiful ?

### DISCUSSION THREAD

**Discussion by laem** [organicmaps/organicmaps#7498] [Q&A]
Hi, dense transit networks are now displayed in a very nice ordered way. Lines that go along on the same paths are displayed in a clean manner. 

As Transit's famous blog post explained. https://blog.transitapp.com/how-we-built-the-worlds-prettiest-auto-generated-transit-maps-12d0c6fa502f/

Is there some open source code for this ? Could it be used in web maps, by producing ckean geojsons from GTFS shapes ? 

Regards

![Screenshot_20240301-105357_Organic Maps](https://github.com/organicmaps/organicmaps/assets/1177762/9a974597-6d6e-41e7-bb72-b13c005373aa)

**Comment by patepelo** [comment]
This might be what your are looking for, or at least part of it: https://github.com/organicmaps/subways

**Reply by laem** [reply]
Thanks, meanwhile I also found https://github.com/ad-freiburg/loom
